arangod/Agency/Supervision.cpp

```diff
@@ -1790,6 +1790,8 @@ bool arangodb::consensus::cleanupFinishedOrFailedJobsFunctional(
   // job), or else the larger job can no longer detect any failures!
   // Returns true if there is something to do, false otherwise.
 
+  using namespace std::literals::string_literals;
```

Comment:
As far as I can see this line is unnecessary.

Reply:
yes no longer.